src/Microsoft.Extensions.Configuration.Abstractions/IConfiguration.cs

```diff
@@ -35,6 +35,10 @@ public interface IConfiguration
         /// <returns>The configuration sub-sections.</returns>
         IEnumerable<IConfigurationSection> GetChildren();
 
+        /// <summary>
+        /// Returns a <see cref="IChangeToken"/> that can be used to obseve when this configuration is reloaded.
+        /// </summary>
+        /// <returns></returns>
```

Comment:
Not sure we should have empty `returns` sections. I would like to ask @bricelam who has some experience with how this works.